endpoints/crab.py

```diff
@@ -3,29 +3,42 @@
 from flask import send_file, after_this_request
 
 from utils.endpoint import Endpoint, setup
+from utils.exceptions import BadRequest
 
 from moviepy.editor import VideoFileClip, TextClip, CompositeVideoClip
 
 
+
 @setup
 class Crab(Endpoint):
-    """This endpoint returns an MP4 file. Make sure your application knows how to handle this format."""
+    """
+    This endpoint returns an MP4 file. Make sure your application knows how to handle this format.
+    This endpoint has a ratelimit of 1 request per 30 seconds.
+    This ratelimit will also apply to all other (future) video based endpoints.
+    Erronous requests count against your limit.
+    Separate text with a comma.
+    """
     params = ['text']
 
     def generate(self, avatars, text, usernames, kwargs):
         @after_this_request
         def remove(response):
-            os.remove(name)
+            try:
+                os.remove(name)
+            except Exception:
+                pass
             return response
-
         t = text.upper().replace(', ', ',').split(',')
+        t_1 = t[0]
+        t_2 = t[1]
+        if (not t_1 and not t_1.strip()) or (not t_2 and not t_2.strip()):
```

Comment:
The `strip`s can be removed with the addition of the above two changes, so this line can be changed to `if not t_1 or not t_2:`